demos/gfql/gfql_cpv_gpu_enchmark.ipynb

```diff
@@ -0,0 +1,2976 @@
+              "      border-bottom-color: var(--fill-color);\n",
+              "    }\n",
+              "    90% {\n",
+              "      border-color: transparent;\n",
+              "      border-bottom-color: var(--fill-color);\n",
+              "    }\n",
+              "  }\n",
+              "</style>\n",
+              "\n",
+              "  <script>\n",
+              "    async function quickchart(key) {\n",
+              "      const quickchartButtonEl =\n",
+              "        document.querySelector('#' + key + ' button');\n",
+              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
+              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
+              "      try {\n",
+              "        const charts = await google.colab.kernel.invokeFunction(\n",
+              "            'suggestCharts', [key], {});\n",
+              "      } catch (error) {\n",
+              "        console.error('Error during call to suggestCharts:', error);\n",
+              "      }\n",
+              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
+              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
+              "    }\n",
+              "    (() => {\n",
+              "      let quickchartButtonEl =\n",
+              "        document.querySelector('#df-857b100f-363e-41f4-8e2a-0628457e85bc button');\n",
+              "      quickchartButtonEl.style.display =\n",
+              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "    })();\n",
+              "  </script>\n",
+              "</div>\n",
+              "    </div>\n",
+              "  </div>\n"
+            ],
+            "application/vnd.google.colaboratory.intrinsic+json": {
+              "type": "dataframe",
+              "summary": "{\n  \"name\": \"fg\",\n  \"rows\": 5,\n  \"fields\": [\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 1,\n        \"min\": 0,\n        \"max\": 4,\n        \"samples\": [\n          1,\n          4,\n          2\n        ],\n        \"num_unique_values\": 5,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
+            }
+          },
+          "metadata": {},
+          "execution_count": 8
+        }
+      ],
+      "source": [
+        "fg = graphistry.edges(df, 's', 'd').materialize_nodes()\n",
+        "print(fg._nodes.shape, fg._edges.shape)\n",
+        "fg._nodes.head(5)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "with 2 and 5 hop `chain` comparison we see a slight/negligable speedup enabled by setting g. to `cudf`"
+      ],
+      "metadata": {
+        "id": "2gVDho9cn2Et"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "for n_hop in [2,5]:\n",
```

Comment:
can we get the output in a clearer table, and a clearer description in the preceeding text of what this is testing?

e.g.,

"Benchmark how using fewer predicates and GPUs speeds up the same task"

+

**Cumulative runtime (s) over 100 iterations**
```
| query | cpu (s) | gpu (s) |
| <query1> | ...
| <query12 | ...
```